packages/flutter_test/lib/src/finders.dart

```diff
@@ -284,6 +285,8 @@ abstract class Finder {
   /// matched by this finder.
   Finder get last => new _LastFinder(this);
 
+  Finder hitTestable(HitOffsetCallback hitOffsetCallback) => new _HitTestableFinder(this, hitOffsetCallback);
```

Comment:
I'm not sure how I can express "center" or "top-left corner" using a finder. I'd like to express "find only those widgets that are able receive hit test requests at a certain location, even if the result of hitTest() is false". The `hitOffsetCallback` produces this "certain location" as an `Offset`. It is a callback because `Offset` is different for each element.

The callback takes a `Finder` because our standard way of getting an `Offset` in the test framework is to use one of `getCenter`, `getTopLeft`, `getBottomRight` methods from the `WidgetController` class, all of which operate on `Finder` objects.

Example:

```dart
// The following says "find widgets with text "foo" able to receive
// hit test check at their center"
controller.tap(find.text('foo').hitTestable(controller.getCenter));
// This one does the same, but interested in top-left corner hit testing
controller.tap(find.text('foo').hitTestable(controller.getTopLeft));
```